### PR TITLE
Issue1294 dynamic icons

### LINKED ISSUE #1294: revise valves and pump icons to indicate how they are controlled
This issue is to implement different graphics to indicate if valves are open/closed and what speed the pumps are operating at. This addresses https://github.com/ibpsa/project1/wiki/2020-01-20-library-development#new-action-items

### PR DESCRIPTION
**Pull request by mwetter**
This closes #1294 
More animations may be added later, but this addresses the most frequently used models for which animation is useful.

### CODE REVIEW

**Comment by mwetter** [comment]
@Mathadon : Can you please review this PR and merge if OK.

File: IBPSA/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo [hunk 1]
```diff
@@ -290,7 +290,8 @@ equation
         Rectangle(
           extent={{-80,60},{80,-60}},
           lineColor={0,0,0},
-          fillColor={95,95,95},
+          fillColor=DynamicSelect({95,95,95},
+          min(1, -Q_flow/Q_flow_nominal)*{255,0,0}+(1-min(1, -Q_flow/Q_flow_nominal))*{95,95,95}),
```
Comment (Mathadon) [inline IBPSA/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo]: Not sure what happens when `Q_flow>Q_flow_nominal` and consequently the color > 255. I presume that it simply saturates to the maximum and that this is fine.

File: IBPSA/Fluid/Sources/PropertySource_T.mo [hunk 2]
```diff
@@ -71,6 +71,12 @@ This model has no dynamics.
 revisions="<html>
 <ul>
 <li>
+February 25, 2020, by Michael Wetter:<br/>
```
Comment (Mathadon) [inline IBPSA/Fluid/Sources/PropertySource_T.mo]: This model only has a modified revision history and no actual changes.

File: IBPSA/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo (hunk 1 above)
Comment (mwetter) [reply IBPSA/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo]: Good point, I will correct this.

File: IBPSA/Fluid/Sources/PropertySource_T.mo (hunk 2 above)
Comment (mwetter) [reply IBPSA/Fluid/Sources/PropertySource_T.mo]: This will be removed.

**Comment by mwetter** [comment]
This is ready to merge when the tests pass.

**Pull request merged**